lib/oli_web/live/progress/resource_title.ex

```diff
@@ -11,13 +11,15 @@ defmodule OliWeb.Progress.ResourceTitle do
 
   def render(assigns) do
     length = length(assigns.node.ancestors)
+    numbering = assigns.node.numbering
 
     ~F"""
     <div>
       <div>
         <small class="text-muted">
           {#for {ancestor, index} <- Enum.with_index(@node.ancestors)}
-            <span>{Oli.Resources.Numbering.container_type(ancestor.section_resource.numbering_level)} {ancestor.section_resource.numbering_index}</span>
+            {n = %Oli.Resources.Numbering{numbering | level: ancestor.section_resource.numbering_level, index: ancestor.section_resource.numbering_index}}
```

Comment:
According to the liveview docs, it is bad practice to assign variables in the template due to the way phoenix does change tracking, I'm not sure but I imagine its probably similar for surface views https://hexdocs.pm/phoenix_live_view/assigns-eex.html#pitfalls